<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>871</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>461</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2b3038;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>321</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #20242a;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(212, 192, 169);</string>
    </property>
    <property name="text">
     <string>&lt;strong&gt;LOG &lt;/strong&gt;IN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #ff662b;
color: rgb(212, 192, 169);
paddinf-bottom: 7px;</string>
    </property>
    <property name="placeholderText">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #ff662b;
color: rgb(212, 192, 169);
paddinf-bottom: 7px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:  rgb(212, 192, 169);
color: #20242a;
border-radius: 10px;
}
QPushButton:hover{
background-color: #837c73;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:  rgb(212, 192, 169);
color: #20242a;
border-radius: 10px;
}
QPushButton:hover{
background-color: #837c73;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>Register Now</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(212, 192, 169);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;You don't have an account ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: red;

QPushButton{
background-color:  transparent;
color: red;;
border-radius: 10px;
}
QPushButton:hover{
background-color: #c40808;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="forgotPasswordButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(212, 192, 169);

QPushButton{
background-color:  transparent;
color: rgb(212, 192, 169);
border-radius: 10px;
}
QPushButton:hover{
background-color: #c40808;
}

QPushButton:pressed{
padding-left: 5px;
padding-right: 5px;
background-position: calc(100% - 10px)center;
background-color: #ff662b;;
}</string>
    </property>
    <property name="text">
     <string>Forgot password?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #272c34;</string>
    </property>
    <property name="text">
     <string>&lt;strong&gt;KASTA &lt;/strong&gt;VA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #272c34;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Virtual Assistant&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/logo/kaastavector.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../icons/res.qrc">:/logo/kaastavector.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../icons/res.qrc"/>
 </resources>
 <connections/>
</ui>
